website/versioned_docs/version-7.1.0/traverse.md

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
You don't need to add this; all versions of 7.x fall back to the 7.0 docs if there's no file in this directory